src/pynxtools_raman/witec/txt/witec_data_model.py

```diff
@@ -0,0 +1,70 @@
+# Copyright The NOMAD Authors.
```

Comment:
You _can_ do it like this, but there is no obligation to do so. This just helps me with type checking in XPS, but you don't have to implement it like this.